k8s/chart/templates/daily-calendar-update-cron.yml

```diff
@@ -0,0 +1,78 @@
+{{- if and .Values.reaper.enable (not .Values.debug) }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "chart.fullname" . }}-calendar_update
```

Comment:
This is so picky I know, but we dont have `_` characters in any of the k8s resource names. Can yall switch this (and all the other `calendar_update`) to `calendar-update`.